### PR TITLE
Removed cargo shuttles in favour of drop pods.

### PR DESCRIPTION
**Pull request by Zuhayr**
- Make order on supply console.
- Confirm order via button or Confirm verb.
- Take chit.
- Find a supply beacon or print one out from the heavy fabricators.
- Take supply beacon a fair way away from the station/ship.
- Insert chit into beacon.
- Wrench beacon.
- Activate beacon.
- Run away and wait.
- Go back, open pod, collect supplies.

### CODE REVIEW

File: code/game/objects/structures/plasticflaps.dm [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		return 0
+
+	if(istype(A, /obj/vehicle))	//no vehicles
+		return 0
+
+	var/mob/living/M = A
+	if(istype(M))
+		if(M.lying)
+			return ..()
+		for(var/mob_type in mobs_can_pass)
```
Comment (Lohikar) [inline code/game/objects/structures/plasticflaps.dm]: `is_type_in_list()`?

Comment (Zuhayr) [reply code/game/objects/structures/plasticflaps.dm]: I moved a bunch of code around, wasn't planning on refactoring any of it.

Comment (Lohikar) [reply code/game/objects/structures/plasticflaps.dm]: Fair enough.

**Comment by Zuhayr** [comment]
This is ready for merge.
https://cdn.discordapp.com/attachments/150453370681950209/357416859773042688/unknown.png

**Pull request merged**